.repo-metadata.json

```diff
@@ -2,7 +2,7 @@
   "name": "rcloadenv",
   "name_pretty": "Google Cloud Runtime Config",
   "product_documentation": "https://cloud.google.com/deployment-manager/runtime-configurator/",
-  "release_level": "ga",
+  "release_level": "beta",
```

Comment:
turns out this uses a beta API

Reply:
boo, we're going in the wrong direction 😆